ovos_utils/configuration.py

```diff
@@ -129,27 +159,27 @@ def get_config_locations(default=True, web_cache=True, system=True,
     if system:
         locs.append(f"/etc/{ovos_cfg['base_folder']}/{ovos_cfg['config_filename']}")
     if web_cache:
-        locs.append(f"{XDG.xdg_config_home}/{ovos_cfg['base_folder']}/web_cache.json")
+        locs.append(get_webcache_location())
     if old_user:
         locs.append(f"~/.{ovos_cfg['base_folder']}/{ovos_cfg['config_filename']}")
     if user:
         if is_using_xdg():
-            locs.append(f"{XDG.xdg_config_home}/{ovos_cfg['base_folder']}/{ovos_cfg['config_filename']}")
+            locs.append(f"{get_xdg_config_save_path()}/{ovos_cfg['config_filename']}")
         else:
             locs.append(f"~/.{ovos_cfg['base_folder']}/{ovos_cfg['config_filename']}")
     return locs
 
 
 def get_webcache_location():
```

Comment:
Potentially out of scope, but this method isn't as self-explanatory as the others; is this a config cache file? Consider adding a docstring

Reply:
calling this out of scope, will address in follow up PR improving config in general  (less logs + yaml support)